clients/fossid-webapp/src/main/kotlin/FossIdRestService.kt

```diff
@@ -159,19 +163,13 @@ interface FossIdRestService {
         fun create(url: String, client: OkHttpClient? = null): FossIdServiceWithVersion {
             logger.info { "The FossID server URL is $url." }
 
-            val minReadTimeout = Duration.ofSeconds(60)
-            val retrofit = Retrofit.Builder()
-                .apply {
-                    client(
-                        client?.run {
-                            takeUnless { readTimeoutMillis < minReadTimeout.toMillis() }
-                                ?: newBuilder().readTimeout(minReadTimeout).build()
-                        } ?: OkHttpClient.Builder().readTimeout(minReadTimeout).build()
-                    )
-                }
-                .baseUrl(url)
-                .addConverterFactory(JacksonConverterFactory.create(JSON_MAPPER))
-                .build()
+            val retrofit = Retrofit.Builder().apply {
+                client(
+                    client?.run {
+                        newBuilder().addInterceptor(TimeoutInterceptor).build()
```

Comment:
Does it make sense to de-duplicate the `Should the `.addInterceptor(TimeoutInterceptor)` call?